Tests/Unit/DataProcessing/MediaConsentProcessorTest.php

```diff
@@ -36,7 +36,7 @@ public function processReturnsProcessedDataWithWrapperActiveWhenMediaConsentItem
         ];
 
         $processor = new MediaConsentProcessor();
-        $result = $processor->process($cObj, [], ['reloadPageType' => 123], $processedData);
+        $result = $processor->process($cObj, [], ['cnProviders' => '', 'reloadPageType' => 123], $processedData);
```

Comment:
Providers is a string? I expected a some kind of array

Reply:
No, its a comma separated string